examples/pet/schema_list.txt

```diff
@@ -0,0 +1,20 @@
+query {
```

Comment:
Meant to do this, appreciate you creating this.

Please move this up a level out of the Pet directory as it applies to all examples.